rules-reviewed/eap6/java-ee/java-ee-jaxrpc.windup.xml

```diff
@@ -22,14 +22,22 @@
                     <location>INHERITANCE</location>
                 </javaclass>
             </when>
-            <perform>
-                <classification title="JAX-RPC Generic Handler" effort="1" severity="mandatory">
-                    <link href="http://docs.oracle.com/cd/E17802_01/webservices/webservices/docs/1.6/api/javax/xml/rpc/handler/GenericHandler.html"
-                        title="Java EE RPC Generic Handler API" />
-                    <tag>jax-rpc</tag>
-                    <tag>soap</tag>
-                </classification>
-            </perform>
-        </rule>
+              <perform>
+                  <hint title="JAX-RPC GenericHandler usage" effort="0" severity="optional">
```

Comment:
I think we haven't used markdown there just because the source report wasn't designed to accomodate it. That could be changed, I think, but I'm not sure what the source report should look like with large hints?